packages/x-data-grid/src/locales/ptPT.ts

```diff
@@ -34,19 +34,19 @@ const ptPTGrid: Partial<GridLocaleText> = {
   // Export selector toolbar button text
   toolbarExport: 'Exportar',
   toolbarExportLabel: 'Exportar',
-  toolbarExportCSV: 'Baixar como CSV',
+  toolbarExportCSV: 'Descarregar como CSV',
   toolbarExportPrint: 'Imprimir',
-  toolbarExportExcel: 'Baixe como Excel',
+  toolbarExportExcel: 'Descarregar como Excel',
 
   // Columns management text
-  // columnsManagementSearchTitle: 'Search',
-  // columnsManagementNoColumns: 'No columns',
-  // columnsManagementShowHideAllText: 'Show/Hide All',
-  // columnsManagementReset: 'Reset',
+  columnsManagementSearchTitle: 'Procurar',
+  columnsManagementNoColumns: 'Sem colunas',
+  columnsManagementShowHideAllText: 'Mostrar/Ocultar Todas',
+  columnsManagementReset: 'Repor',
```

Comment:
I believe `columnsManagementReset` is omitted for all locales.

Reply:
Yes, the `columnsManagementReset` key is supposed to be translated just like every other.